### ISSUE TITLE
The End of Journey

### ISSUE THREAD

**Issue by Sangwan5688** [Sangwan5688/BlackHole#344]
So back in Jan 2021, I started working on an app just to listen to and download songs on my mobile. It was the first project I put my heart into. I was still a college student then, but a month ago, I graduated and started a job, so I’m unable to give enough time to BlackHole. Sorry for making you guys wait for updates coz of that. I just wanted to create a perfect music app for all needs, mine and yours. But, I was never able to achieve that; I guess I still lag a lot. Anyways, It’s been a long journey for me. I met many incredible people, made new friends, and even saw people admire my work. Who would have guessed that someone would admire my work? But every journey has an end, and this is the end of mine. I’m planning to stop the development of BlackHole furthermore. I know there are still a lot of bugs and missing features; sorry :( for all that. I will try releasing a version once in a while consisting of the changes pushed by the open-source community (PS: PRs welcomed), but there won’t be further development from my side. Thanks for your love and support, and most importantly, for using BlackHole.
I hope you all enjoyed it :)

**Comment by gmongs** [comment]
Thank you very much.  Always be happy :)

**Comment by BrightDV** [comment]
You have done such an amazing music app!
Thank you for all your work!
(And with this app I made my first PRs :))

**Comment by Lavin-tom** [comment]
@Sangwan5688 Thank you so much and All the very best,

**Comment by agam778** [comment]
No problem, and thanks for the great work you've done for us! All the best for your life

**Comment by topcheese** [comment]
Yeah and I just decided to make this my main music player a couple of days ago.  It's nothing on you, but just my  usual luck.  Thank you so much for the time and effort put into it, and may your new journey bring you much joy and happiness ... as you have brought to most all of us with your project!

**Comment by RafeeDaBoy** [comment]
I just found out about this recently and has been my main app since then. It is absolutely amazing and I'm really grateful for the work that you've done. I'm sad to hear the development is going to stop, but it is already near perfect. Thank you so much!

**Comment by yarons** [comment]
Hi, first of all thank you very much for all your incredible work.

Is it possible to add some comments to the Weblate project so the translators won't work for translation that won't be committed eventually?

**Comment by iPranshu11** [comment]
ಠ_ಠ

**Comment by iamanuragmondal** [comment]
Please don't end this project 🥺🥺🙏🙏

**Comment by 024x** [comment]
Hey there! @Sangwan5688 Bro, Your hard work and dedication to BlackHole is truly appreciated by all of us. We understand your current circumstances, but we want you to know that we are here to continue the project and ensure it reaches its full potential. Your journey may have ended, but we are here to carry it on. Thank you for your awesome application.
Hope you will be great in future, Take Care, Thanks again 😊.

**Comment by Bharadwaj-R** [comment]
Hello, @Sangwan5688. First of all, thanks for creating the best music app out there. It perfectly fits all my needs and I recommended the app to everyone I could think of. Seeing this message gives me a sad face. But all good things end at some point. It is the end of an incredible journey, and I wish you all luck in your future.

**Comment by piyushaswani55** [comment]
Thanks @Sangwan5688 for this amazing app. ❤️ 
I just came across this another app on playstore which seems like a copy of BlackHole, just thought you should know.
https://play.google.com/store/apps/details?id=com.shadow.AlphaPlayer

**Comment by 024x** [comment]
> Thanks @Sangwan5688 for this amazing app. ❤️ I just came across this another app on playstore which seems like a copy of BlackHole, just thought you should know. https://play.google.com/store/apps/details?id=com.shadow.AlphaPlayer


As @Sangwan5688 mentioned in README.md
`Being Open Source doesn't mean you can just make a copy of the app and upload it on playstore or sell
a closed source copy of the same.
Read the following carefully:
1. Any copy of a software under GPL must be under same license. So you can't upload the app on a closed source
  app repository like PlayStore/AppStore without distributing the source code.
2. You can't sell any copied/modified version of the app under any "non-free" license.
   You must provide the copy with the original software or with instructions on how to obtain original software,
   should clearly state all changes, should clearly disclose full source code, should include same license
   and all copyrights should be retained.

In simple words, You can ONLY use the source code of this app for `Open Source` Project under `GPL v3.0` or later
with all your source code CLEARLY DISCLOSED on any code hosting platform like GitHub, with clear INSTRUCTIONS on
how to obtain the original software, should clearly STATE ALL CHANGES made and should RETAIN all copyrights.
Use of this software under any "non-free" license is NOT permitted.`

The person didn't respected any license or anything. It was an issue we should do something.

**Comment by piyushaswani55** [comment]
Yes @024x , that's why I thought of bringing it to attention, I could have understood making a fork or continuing the project as a separate open source application but selling it on playstore is disrespecting both licensing and general ethics. 
I have already reported the application and wanted to bring it to @Sangwan5688's notice since he is the owner, he should be able to take some action on his own.

Also, @Sangwan5688 I understand that time management is tough with a full time job to work on a side project like this but I think it shouldn't be ended. If possible, you can consider opening it to wider contributors and get help from opensource community. You don't come across such wonderful opensource applications on a daily basis. Who knows one day this becomes even more popular than apps like Spotify itself. :)

**Comment by 024x** [comment]
@piyushaswani55  we can also flag it in playstore. 
Go to the link you provided (playstore link)
Click on 3 dots(top right) and flag as Inappropriate. 
Select Copycat or impression. The  submit. 

This my make playstore to take some action or may not.

**Comment by 05789436** [comment]
"I know there are still a lot of bugs and missing features; sorry :( for all that."
You're not sorry, don't lie! 🙄If you really cared about your community, then you would continue developing this app! If you start something, you ought to finish it. Otherwise don't waste people's time!

"I graduated and started a job, so I’m unable to give enough time to BlackHole."
BS! Are you telling us that you have to work 24/24 hours without any free time?¿ XDDD What are you, a robot?¿ Common dude! 👎 Don't treat us like we are children that still believes in Santa, lol.  You're just fed up with this project. Call it the way it is, without BS us.

**Comment by vedxp** [comment]
> Thanks @Sangwan5688 for this amazing app. ❤️ I just came across this another app on playstore which seems like a copy of BlackHole, just thought you should know. https://play.google.com/store/apps/details?id=com.shadow.AlphaPlayer

Found another copycat, I've flagged/reported both:
https://play.google.com/store/apps/details?id=com.vuzeninfotech.musify

@Sangwan5688 please pay attention to this :)

**Comment by Narendravarma123** [comment]
Message ID: ***@***.*** @Sangwan5688 for this amazing antique of yours .It is one of the best music app I have ever seen.And I appreciate your concern and your efforts in building this BlackHole App and your dedication towards in debugging the bugs we encounterd in the app.Once again thank you

**Comment by Ryuxaki** [comment]
@Sangwan5688 
You have made an app for users like me who want to listen to music anytime. I really want to say Thanks from my heart. 

If you make some updates in it and try to improve it more, I promise this app will be more famous and better than Spotify and Wynk. Thank you for your efforts ❤️

**Comment by Ryuxaki** [comment]
Please don't stop this project
@Sangwan5688 

You have made something incredible. Please work on it more, please don't stop this. 😭

**Comment by lepras** [comment]
Thanks for all the work!

I see a great opportunity here, I have been on a hunt for a private music app. I think this can be made into that.

I have forked the repo as "Milkyway" here:

https://github.com/lepras/Milkyway/

And my top-priority goals are as follows:

- [ ] smooth windows builds
- [ ] source from youtube for i18n
- [ ] sync server (self hostable or otherwise)
- [ ] share playlists & songs as spotify, youtube & apple music url's
- [ ] Remove spotify from home page
- [ ] Make the app as private as possible - don't give youtube anything

Note that I will be using only youtube as a primary source as that's available worldwide and remove spotify & jiosaavn as their privacy policy is ... well not private at all. Will tweak such that nothing is given to youtube, probably only IP and that's a non-issue.

**Comment by sdk5168** [comment]
This work is great

**Comment by 6801318d8d** [comment]
@Sangwan5688  who are the current maintainers? Because I see people pushing on this repo, thus having write access to it, which are not you.

Do you plan to hand over to another maintainer?

**Comment by S4tyendra** [comment]
> Damn! just when I came across this beautiful app only to know it won't be worked on further 😭  

Don't worry my bro..
This project will not be discontinued

**Comment by Logan7899** [comment]
Ankit are you back
